testing/run_tests.py

```diff
@@ -1054,6 +1071,24 @@ def main():
         build_dir, engine_filter, args.coverage, args.engine_capture_core_dump
     )
 
+  # Use this type to exclusively run impeller vulkan tests.
+  # TODO (https://github.com/flutter/flutter/issues/113961): Remove this once
+  # impeller vulkan tests are stable.
+  if 'impeller-vulkan' in types:
+    build_name = args.variant
+    xvfb.StartVirtualX(build_name, build_dir)
```

Comment:
Consider making a contextmanager for this. Or at least put the `Stop` call in a `finally`.